### PR TITLE
add copyplugin that copies music

### LINKED ISSUE #107: check configuration - musi doesnt play after build (is not being injected)
check: CopyWebpackPlugin (needed in dist/index.html)

in typescript files (where needed):
`const deafultAudio = require('../../assets/audio/music.mp3')`
then music is being contained in `deafultAudio.default` object property

### PR DESCRIPTION
**Pull request by ibednorz**

### CODE REVIEW

**Review by pdybowski — CHANGES_REQUESTED** [review]
We also need to have audio files in our dist file because our code have different path to get audio then /assets/audio. Please add const music_name = require('path') in files where we use audio. Then change src path from string to music_name

File: src/app/components/battle.ts [hunk 1]
```diff
@@ -2,6 +2,9 @@ import { createElement } from '../utils/createElement'
 import { Audio } from '../audio'
 import { Game } from '../game'
 
+const music = require('../../assets/audio/music.mp3')
+const battle = require('../../assets/audio/battle.mp3')
```
Comment (pdybowski) [inline src/app/components/battle.ts]: Rename it to battle_music

File: src/app/components/battle.ts [hunk 2]
```diff
@@ -2,6 +2,9 @@ import { createElement } from '../utils/createElement'
 import { Audio } from '../audio'
 import { Game } from '../game'
 
+const music = require('../../assets/audio/music.mp3')
```
Comment (pdybowski) [inline src/app/components/battle.ts]: Rename it to main_music or deafult_music

File: webpack.common.js [hunk 3]
```diff
@@ -50,5 +51,11 @@ module.exports = {
       filename: '[name].css',
       chunkFilename: '[id].css',
     }),
+    new CopyPlugin({
+      patterns: [
+        { from: "src/assets/audio", to: "assets/audio" },
+        { from: "src/assets/audio", to: "" }
```
Comment (pdybowski) [inline webpack.common.js]: I think we don't need that.when we have require(something) in scripts. Webpack 'pack' music in the dist directory

**Review by pdybowski — CHANGES_REQUESTED** [review]

**Review by hoolek77 — APPROVED** [review]

**Review by pdybowski — APPROVED** [review]

**Pull request merged**